lightbulb-challenge-sammy/src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import LightSwitch from './components/LightSwitch';
+import './App.css';
+
+// Display component only
+const App = () => {
+  return (
+    <>
+      <h1>Lightbulb Challenge</h1>
+      <h2>Turn the lights on or off, by clicking any box.</h2>
```

Comment:
Good instruction to the user.

Reply:
I agree with James. Instructions are always good

Reply:
I also like that you added instructions for the user.